### PR TITLE
[17.0][MIG] hr_timesheet_task_stage: Migration to 17.0

### PR DESCRIPTION
**Pull request by RogerSans**
Supersede = https://github.com/OCA/timesheet/pull/650 
Standard migration
T-6450

@HaraldPanten 
@ValentinVinagre 
@luis-ron

### CODE REVIEW

File: hr_timesheet_task_stage/views/account_analytic_line.xml [hunk 1]
```diff
@@ -17,15 +17,15 @@
                     tabindex="-1"
                     type="object"
                     icon="fa-folder-o"
-                    attrs="{'invisible': ['|', ('is_task_closed', '=', True), ('task_id', '=', False)]}"
+                    invisible="is_task_closed == True or task_id == False"
```
Comment (alexey-pelykh) [inline hr_timesheet_task_stage/views/account_analytic_line.xml]: `invisible="is_task_closed or not task_id"` is sufficient

File: hr_timesheet_task_stage/views/account_analytic_line.xml [hunk 2]
```diff
@@ -17,15 +17,15 @@
...
                 />
                 <button
                     name="action_open_task"
                     title="Open task"
                     tabindex="-1"
                     type="object"
                     icon="fa-folder-open-o"
-                    attrs="{'invisible': ['|', ('is_task_closed', '=', False), ('task_id', '=', False)]}"
+                    invisible="is_task_closed == False or task_id == False"
```
Comment (alexey-pelykh) [inline hr_timesheet_task_stage/views/account_analytic_line.xml]: `invisible="not is_task_closed or not task_id"`

File: hr_timesheet_task_stage/views/account_analytic_line.xml [hunk 3]
```diff
@@ -40,7 +40,7 @@
                     <field name="is_task_closed" invisible="1" />
                     <button
                         name="action_toggle_task_stage"
-                        attrs="{'invisible': [('is_task_closed', '=', True)]}"
+                        invisible="is_task_closed == True"
```
Comment (alexey-pelykh) [inline hr_timesheet_task_stage/views/account_analytic_line.xml]: `invisible="is_task_closed"`

File: hr_timesheet_task_stage/views/account_analytic_line.xml [hunk 4]
```diff
@@ -51,7 +51,7 @@
                     </button>
                     <button
                         name="action_toggle_task_stage"
-                        attrs="{'invisible': [('is_task_closed', '=', False)]}"
+                        invisible="is_task_closed == False"
```
Comment (alexey-pelykh) [inline hr_timesheet_task_stage/views/account_analytic_line.xml]: `invisible="not is_task_closed"`

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]
Few minor requests

File: hr_timesheet_task_stage/views/account_analytic_line.xml (hunk 4 above)
Comment (RogerSans) [reply hr_timesheet_task_stage/views/account_analytic_line.xml]: THNX for review. Changed.

File: hr_timesheet_task_stage/views/account_analytic_line.xml (hunk 3 above)
Comment (RogerSans) [reply hr_timesheet_task_stage/views/account_analytic_line.xml]: THNX for review. Changed.

File: hr_timesheet_task_stage/views/account_analytic_line.xml (hunk 2 above)
Comment (RogerSans) [reply hr_timesheet_task_stage/views/account_analytic_line.xml]: THNX for review. Changed.

File: hr_timesheet_task_stage/views/account_analytic_line.xml (hunk 1 above)
Comment (RogerSans) [reply hr_timesheet_task_stage/views/account_analytic_line.xml]: THNX for review. Changed.

**Review by alexey-pelykh — APPROVED** [review]
Code LGTM

**Comment by HaraldPanten** [comment]
@pedrobaeza The original PR seems to be abandoned. Do you mind closing the other and assign this migration to us?

THX in advance!

**Review by luis-ron — APPROVED** [review]
Functional Review: LGTM 👍🏻

**Comment by RogerSans** [comment]
@manuelregidor can you review this pls?

**Review by HaraldPanten — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**